webview/src/experiments/components/table/header/TableHeaderCell.tsx

```diff
@@ -51,16 +45,11 @@ const getHeaderPropsArgs = (
         [styles.paramHeaderCell]: columnWithGroup.group === ColumnType.PARAMS,
         [styles.metricHeaderCell]: columnWithGroup.group === ColumnType.METRICS,
         [styles.depHeaderCell]: columnWithGroup.group === ColumnType.DEPS,
-        [styles.createdHeaderCell]: header.id === 'Created',
-        [styles.firstLevelHeader]: isFirstLevelHeader(header.column.id),
-        [styles.leafHeader]: header.subHeaders === undefined,
-        [styles.menuEnabled]: sortEnabled,
-        [styles.sortingHeaderCellAsc]: sortOrder === SortOrder.ASCENDING,
-        [styles.sortingHeaderCellDesc]:
-          sortOrder === SortOrder.DESCENDING && !header.isPlaceholder,
-        [styles.oneRowHeaderCell]: onlyOneLine
-      },
-      headerDropTargetId === header.id && styles.headerCellDropTarget
+        [styles.firstLevelHeaderCell]: isFirstLevelHeader(header.column.id),
+        [styles.leafHeaderCell]: header.subHeaders === undefined,
+        [styles.oneRowHeaderCell]: onlyOneLine,
+        [styles.dropTargetHeaderCell]: headerDropTargetId === header.id
```

Comment:
Since all of these object classes accomplish the same thing (applying extra styles to `headerCell/placeHolderCell`), I renamed them to all have the `HeaderCell` at the end, basing that off the first five.